src/placeos-rest-api/utilities/current-user.cr

```diff
@@ -18,7 +18,7 @@ module PlaceOS::Api
       end
 
       # check for X-API-Key use
-      if (token = request.headers["X-API-Key"]?)
+      if (token = request.headers["X-API-Key"]? || params["api-key"]?)
```

Comment:
Would it be worth throwing in a `presence` here?

```suggestion
      if (token = request.headers["X-API-Key"]?.presence || params["api-key"]?.presence)
```

Reply:
I don't think we need `presence` adds processing overhead and the header / param shouldn't be set empty